### ISSUE TITLE
Table d3plus v2

### ISSUE THREAD

**Issue by ranucame** [alexandersimoes/d3plus#659]
Hello,
Is it possible to make a table with d3plus, which includes features like colSpan and rowSpan? 

Thanks

**Comment by davelandry** [comment]
No, the current implementation of d3plus v2 does not have the table visualization type. we decided to not include it, as basic table functionality is generally better handled by HTML or front-end frameworks like React. Is there a reason you'd specifically like to use JavaScript?
